src/lib/validate-input.js

```diff
@@ -471,7 +471,10 @@ export function validatePool2dParams(input, _, {roundingType = 'floor'}) {
   }
 }
 
-export function validateReduceParams(input, {axes}) {
+export function validateAxes(input, {axes}) {
```

Comment:
Given the renaming, should it just take `axes` argument rather than an object containing `axes` attribute?